### ISSUE TITLE
AV: A11y HCM issue with Guidance in psammead-media-player

### ISSUE THREAD

**Issue by HarveyPeachey** [BBC-archive/psammead#2572]
**Is your feature request related to a problem? Please describe.**
In the dev a11y swarm done for #2517, a few issues were found as documented in this [comment](https://github.com/bbc/psammead/pull/2517#issuecomment-549826195):

One of these issues is with Windows HCM. Currently, with the implementation of guidance, the current opaque overlay is causing the placeholder image to not be visible which it should be in High Contrast Mode.

**Describe the solution you'd like**
Our implementation for the overlay is using `background-color: rgba(34,34,34,0.75);`. From a brief bit of investigation, it looks like `background-color` in Windows HCM is being replaced by a block colour hence the placeholder image not appearing. 

We need to find a way to implement the opaque layer without blocking the placeholder image behind it in HCM.

**Describe alternatives you've considered**

**Testing notes**
[Tester to complete]

Dev insight: Will there be any potential regression? etc

- [x] This feature is expected to need manual testing.

**Additional context**
